config/routes.rb

```diff
@@ -191,6 +194,9 @@
   end
   resources :case_court_orders, only: %i[destroy]
 
+  resources :additional_expenses, only: %i[create destroy],
+    constraints: lambda { |req| req.format == :json }
```

Comment:
I did these as base resources, when they could be member resources, mostly because it was easier. Can change to member resources if needed. Although in a new ticket would be nice.

The constraint for :json was new to me, but I thought it made sense.